Gems/AWSGameLift/cdk/README.md

```diff
@@ -18,7 +19,11 @@ you can create the virtualenv manually.
 See https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html about for information about how to set up
 the prerequisites for CDK development.
 
-To manually create a virtualenv on MacOS and Linux:
+
+>Note: This stack is for CDK v2 (the latest CDK version, if you are working with CDKv1 stacks please use the CDKv1 version of this application).
```

Comment:
nit: missing space between `>` and "Note:...`

Reply:
Will fix